tests/test_qpax.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright 2016-2024
```

Comment:
```suggestion
# Copyright 2024 Lev Kozlov
```

The copyright is just a disclaimer (it applies whether we write it or not, so it's just written as an indication). If you write code as an individual your name goes there, and if you work for an institution your institute's name goes there (for instance I work for Inria so the code I write today is "Copyright 2024 Inria").